build-aux/snap/snapcraft.yaml

```diff
@@ -466,6 +471,9 @@ parts:
         touch "${CRAFT_PART_INSTALL}/fips-build-lp"
       fi
 
+      if [ "${CRAFT_ARCH_BUILD_FOR}" = "arm64" ]; then
+        cp -a --parents /usr/lib/${CRAFT_ARCH_TRIPLET_BUILD_FOR}/libteec.so* "${CRAFT_PART_INSTALL}/"
+      fi
```

Comment:
The `optee-client` package should be staged instead, I think that in the `runtime` part